lib/media/drm_engine.js

```diff
@@ -512,7 +512,8 @@ shaka.media.DrmEngine = class {
     // as keys in a dictionary.
     const metadatas = this.activeSessions_.values();
     for (const metadata of metadatas) {
-      if (shaka.util.BufferUtils.equal(initData, metadata.initData)) {
+      if (shaka.util.BufferUtils.equal(initData, metadata.initData)
+       && !shaka.util.Platform.isTizen2()) {
```

Comment:
It seems strange to deliberately create duplicate sessions on just one platform.  I believe you that it's necessary, but this definitely needs a comment explaining why we're doing it.

Also, a minor style issue, but please indent this line 4 spaces more than the "if" above.  You can run ./build/check.py to do a minimal linter & compiler check.